Пакулин Сергей

IIaKyJIuH opened the pull request:
Но требует ещё доработки в классах css
[Посмотреть решение](https://hruview.herokuapp.com/IIaKyJIuH/markup-task-5/master/index.html)

IIaKyJIuH committed:
Готов

Но требует ещё доработки в классах css

honest-hrundel commented:
:green_apple: Пройден линтинг и базовые тесты

IIaKyJIuH committed:
все изображения

honest-hrundel commented:
:green_apple: Пройден линтинг и базовые тесты

IIaKyJIuH committed:
Незначительные правки

honest-hrundel commented:
:green_apple: Пройден линтинг и базовые тесты

IIaKyJIuH committed:
Добавил выравнивание картинки по центру

honest-hrundel commented:
:green_apple: Пройден линтинг и базовые тесты

mokhov commented on index.css:
```diff
@@ -0,0 +1,462 @@
+header
+{
+    font: 32px 'PT Sans Narrow';
+    position: fixed;
```
Это можно сделать без position: fixed

mokhov reviewed (changes requested):
1. При изменении высоты экрана футер наползает на котов
2. В попапе есть элементы, на которые можно навестись, но нельзя кликнуть
3. Затенение показывается поверх шапки, но не футера
4. Хорошо бы не портить пропорции картинок
🍅

IIaKyJIuH committed:
Исправил

В header нет теперь position: fixed
1. Не уверен, что сделал правильно, но просто сделал футеру background, чтобы не сливался с котами при изменении высоты окна.
2. Убрал из попапа все элементы с наводкой, на которые нельзя было нажать.
3. Затенение теперь есть и у шапки, и у футера.
4. Постарался максимально сохранить пропорции картинок как в самой карусели, так и в попапе

honest-hrundel commented:
:green_apple: Пройден линтинг и базовые тесты

IIaKyJIuH committed:
Доработал футер

+ещё мелки правки

honest-hrundel commented:
:green_apple: Пройден линтинг и базовые тесты

mokhov commented:
Вот это
> Затенение показывается поверх шапки, но не футера

Не исправил
🍅
